### ISSUE TITLE
Having an odd issue loading the mod.

### ISSUE THREAD

**Issue by Naeluu** [0Mayall/ONIMods#2]
Hello.
I'm getting the error attached as an image.
![Untitled](https://user-images.githubusercontent.com/59373555/71598844-cb7fa000-2b48-11ea-9ccc-953cf9a5c950.png)
I'm not sure what to do. There is over 3TB of free space on this drive, the entire 'Klei' folder in this path has full control access to the group 'everyone', and this is the only mod this occurs.
If you need more info, tell me what I need to do to get you more info.

**Comment by 0Mayall** [comment]
Hi, the last time someone had this issue it was because his hard drive was failing. I do not know why it is failing for you.

**Comment by Naeluu** [comment]
I doubt that my hardrive is failing, I use it for a lot of different things without fail.
I also doubt this because originally the error message was pointing to a onedrive folder, and I assumed the error was coming from that. So I changed my libraries to be on my Bulk hardrive, but the error persisted.
I could try changing libraries again. Let me get back to you on that.

**Comment by 0Mayall** [comment]
Yeah, I misspoke a little there. I meant I do not know why it's failing to write to your disk, not I do not know why your hard drive is failing.
